### PR TITLE
CMR 3964 - Add validation for the bulk update POST body

### PR DESCRIPTION
**Pull request by laurenfrederick**

### CODE REVIEW

File: common-lib/src/cmr/common/validations/json_schema.clj [hunk 1]
```diff
@@ -59,14 +59,20 @@
       (let [message (str/replace (.getMessage e) #"\[Source[^;]+;" "")]
         (errors/throw-service-error :bad-request (str "Invalid JSON: " message))))))
 
+(defn json-string->json-schema
+ "Convert a string to com.github.fge.jsonschema.main.JsonSchema object."
```
Comment (daniel-zamora) [inline common-lib/src/cmr/common/validations/json_schema.clj]: Why is there only 1 space indent here? Noticing this a lot in the data-center code.

File: ingest-app/src/cmr/ingest/api/ingest.clj [hunk 2]
```diff
@@ -12,7 +15,7 @@
    [cmr.common.services.errors :as srvc-errors]
    [cmr.common.util :as util]
    [cmr.common.xml.gen :refer :all]
-   [cmr.common-app.api.enabled :as common-enabled]
+   [cmr.ingest.services.bulk-update-service :as bulk-update]
```
Comment (daniel-zamora) [inline ingest-app/src/cmr/ingest/api/ingest.clj]: Is this only ever going to be an update service? Or are we going to try other bulk operations?

File: common-lib/src/cmr/common/validations/json_schema.clj (hunk 1 above)
Comment (daniel-zamora) [inline common-lib/src/cmr/common/validations/json_schema.clj]: https://github.com/bbatsov/clojure-style-guide#source-code-layout--organization

File: ingest-app/src/cmr/ingest/api/ingest.clj (hunk 2 above)
Comment (laurenfrederick) [inline ingest-app/src/cmr/ingest/api/ingest.clj]: I have no idea. I imagine there could be other operations down the road but have not seen plans for those.

File: common-lib/src/cmr/common/validations/json_schema.clj (hunk 1 above)
Comment (laurenfrederick) [inline common-lib/src/cmr/common/validations/json_schema.clj]: Fixed

File: ingest-app/src/cmr/ingest/api/ingest.clj [hunk 3]
```diff
@@ -361,8 +364,10 @@
 (defn- bulk-update-collections
```
Comment (daniel-zamora) [inline ingest-app/src/cmr/ingest/api/ingest.clj]: I see one here that needs indenting.  There are probably a lot more from your first merge.  I will approve this merge but I think the others should get fixed eventually too.

**Pull request merged**